sencha/index.py

```diff
@@ -180,7 +201,12 @@ def maybe_save_peptide_bloom_filter(
             peptide_bloom_filter.save(save_peptide_bloom_filter)
         else:
             suffix = f".alphabet-{molecule}_ksize-{ksize}.bloomfilter." f"nodegraph"
```

Comment:
nitpicky but  couldn't the suffix be  f".alphabet-{molecule}_ksize-{ksize}.bloomfilter.nodegraph"? and not your change, so don't worry